File: program/find-sum-of-n-natural-numbers/FindSumOfNNaturalNumbers.java
public class FindSumOfNNaturalNumbers {
    public static void main(String[] args) {
        int num = 8;
        int sum = 0;

        for (int i = 1; i <= num; i++) {
            sum += i;
        }

        System.out.println(sum);
    }
}

File: program/find-sum-of-n-natural-numbers/findSumOfNNaturalNumbers.scala
object HelloWorld {
	def main(args: Array[String]): Unit = {
	  var n = scala.io.StdIn.readInt();

	 //Sum using for loop
   var total = 0;      
   for (i <- 0 to n) {
     total += i;
   }
   println(s"\nSum: ${total}");
	}
}
